.github/workflows/dashboard_deploy_template.yaml

```diff
@@ -0,0 +1,80 @@
+name: Dashboard Deploy
+
+on:
+  workflow_call:
+    inputs:
+      parameter_file:
+        description: "Path to the parameters file"
+        required: true
+        type: string
+
+jobs:
+  deployShiny:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install yq
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y yq
+
+      - name: Read parameters
+        run: |
+          DASHBOARD_NAME=$(yq '.dashboard_name' ${{ inputs.parameter_file }})
+          DEPLOY_TARGET=$(yq '.deploy_target' ${{ inputs.parameter_file }})
+          INSTALL_GDAL=$(yq '.install_gdal' ${{ inputs.parameter_file }})
+          EXTRA_PACKAGES=$(yq '.extra_packages | join(" ")' ${{ inputs.parameter_file }})
+
+          echo "DASHBOARD_NAME=$DASHBOARD_NAME" >> $GITHUB_ENV
+          echo "DEPLOY_TARGET=$DEPLOY_TARGET" >> $GITHUB_ENV
+          echo "INSTALL_GDAL=$INSTALL_GDAL" >> $GITHUB_ENV
```

Comment:
We also sometimes have issues with the units package and on ubuntu need to add extra steps to install dependencies for it. So I think there's an extra option we should add in for INSTALL_UNITS_DEPENDENCIES, which like the current INSTALL_GDAL, is a boolean value that can be set in the yml and then if true, will add an extra step something like this:
```
- name: Install units dependencies
        run: sudo apt-get install libudunits2-dev
```